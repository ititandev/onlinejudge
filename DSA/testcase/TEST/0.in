10
4.5 2.1 3.6 32.3 -6.4 -3.5 64.3 -43.2 43 23.3
