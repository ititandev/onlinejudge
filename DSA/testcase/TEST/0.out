   -43.2    -6.4    -3.5     2.1     3.6
     4.5    23.3    32.3      43    64.3
